framework/src/main/java/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -499,10 +499,11 @@ private void addFinalLocalValues(S store, Element enclosingElement) {
      *
      * @param methodDeclTree the declaration of the method or constructor
      * @return true if the receiver of a method or constructor might not yet be fully initialized
+     * @see GenericAnnotatedTypeFactory#isNotFullyInitializedReceiver(MethodTree)
      */
     @Pure
-    protected boolean isNotFullyInitializedReceiver(MethodTree methodDeclTree) {
-        return TreeUtils.isConstructor(methodDeclTree);
+    protected final boolean isNotFullyInitializedReceiver(MethodTree methodDeclTree) {
+        return analysis.atypeFactory.isNotFullyInitializedReceiver(methodDeclTree);
```

Comment:
Could you change the javadoc to make explicit that this is just calling the ATF method?